src/Microsoft.IdentityModel.JsonWebTokens/JwtTokenUtilities.cs

```diff
@@ -65,50 +63,6 @@ public class JwtTokenUtilities
             JwtHeaderParameterNames.Zip
         };
 
-        /// <summary>
-        /// Creates a dictionary from a list of Claim's.
-        /// </summary>
-        /// <param name="claims"> A list of claims.</param>
-        /// <returns> A Dictionary representing claims.</returns>
-        internal static IDictionary<string, object> CreateDictionaryFromClaims(IEnumerable<Claim> claims)
```

Comment:
What is the reason for moving this method to M.IM.Tokens?

Reply:
`TokenValidationResult  ` is in M.IM.Tokens, and to use JwtTokenUtilities it has to refer to M.IM.JsonWebTokens . Which causes circular dependency.